### PR TITLE
Add CHANGELOG for 1.2.0-rc3

### PR DESCRIPTION
**Pull request by akshaysngupta**
## Checklist
- [x] The title of the PR is clear and informative
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] Issues addressed by the PR are mentioned in the description followed by `Fixes`.

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.2.md [hunk 1]
```diff
@@ -1,11 +1,27 @@
...
+In this release, AGIC will use the new `hostnames` property in HTTP Listener in Application Gateway instead of `hostname`. With this property, We will be able to expose support for Wild Card hostnames with characters like * and ? allowed to match characters.  
+We are working on bringing Azure Portal support for the new property soon. Until those changes arrive, Users will not be able to view the hostname in the listener section on Portal.
+
+## Fixes:
+* [#867][https://github.com/Azure/application-gateway-kubernetes-ingress/issues/867] Set UnhealthyThreshold on Application Gateway to 20 when readiness/liveness probe has UnhealthyThreshold > 20
+* [#876][https://github.com/Azure/application-gateway-kubernetes-ingress/issues/876] Allow using shared feature in both helm 2 and helm 3
+* [#887][https://github.com/Azure/application-gateway-kubernetes-ingress/issues/887] Update shared feature code to use new listener's hostnames during blacklist filtering
+* [#890](https://github.com/Azure/application-gateway-kubernetes-ingress/issues/890) Correct scheme contruction for k8s event recorder
+
 # v1.2.0-rc2
```
Comment (3quanfeng) [inline CHANGELOG/CHANGELOG-1.2.md:25]: Thanks, maybe we could do in a separate PR, but for the release note of rc2, we might need to add one more note regarding the broken part of prohibited target feature and suggest to move to rc3 to use prohibited target feature

Comment (akshaysngupta) [reply CHANGELOG/CHANGELOG-1.2.md:25]: good point. I will add in this PR only.

File: CHANGELOG/CHANGELOG-1.2.md [hunk 2]
```diff
@@ -38,6 +48,13 @@ We are working on bringing Azure Portal support for the new property soon. Until
 * Switch from `hostname` to `hostnames` property in HTTP Listener on Application Gateway.
 * helm: remove replica count setting from supported helm values. This will be added back when AGIC will recieve support for leader election.
 
+## Known Issues:
```
Comment (3quanfeng) [inline CHANGELOG/CHANGELOG-1.2.md:51]: In addition, we have a fix in rc3 to address prohibited target with hostnames, which means rc2 doesn't have the fix...better to notify here in case rc2 is used for shared feature?
